main.py

```diff
@@ -192,6 +204,25 @@ async def compile_project(project_root: Path, manifest: PackageManifest):
         results[project_root.name] = compiled_manifest
         tasks[project_root.name] = TaskStatus.SUCCESS
     except CompilerError as e:
+        breakpoint()
+        # Convert the error details into the Pydantic model
+        error_details = [
+            CompilerErrorModel(
+                message=e.get('message', 'Unknown error'),
+                column=e.get('sourceLocation', {}).get('column', None),
+                line=e.get('sourceLocation', {}).get('line', None),
```

Comment:
by default, the 2nd arg to `.get()` on a `dict` is already `None`, so you don't need this
```suggestion
                column=e.get('sourceLocation', {}).get('column'),
                line=e.get('sourceLocation', {}).get('line'),
```